index.bs

```diff
@@ -3093,6 +3093,43 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+## Authenticator Biometric Criteria Extension (biometricBound) ## {#sctn-authenticator-biometric-criteria-extension}
+
+This [=registration extension=] allows a [=[RP]=] to specify the desired performance of a biometric authenticator
+that could be leveraged when creating the credential. 
+
+: Extension identifier
+:: `biometricBound`
+
+: Client extension input
+:: Biometric performance bounds:
+
+    <pre class="idl">
+    dictionary biometricCriteria{
```

Comment:
perhaps `biometricCriteria` should be `authenticatorBiometricMatchingCriteria` ?